tests/cephfs/BUG-1980920.py

```diff
@@ -0,0 +1,148 @@
+import json
```

Comment:
IMO, the filename could be descriptive. Also, could we avoid using upper case?

It would be good if add a description on the purpose of the test module. In the description, we could add references to the BZ.

Reply:
I have changed the filename.
I also have added the reference to the BZ as well